images/sqlpad/tests/03-helm-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+git clone https://github.com/sqlpad/sqlpad.git && cd sqlpad/sqlpad-charts
+
+# Set up a trap to remove the directory on script exit
+cleanup() {
+    echo "Cleaning up..."
+    rm -rf ../../sqlpad
+    cd ../../
```

Comment:
Using relative paths for `rm -rf` like this looks a bit dangerous to me. What happens if I invoke this script from a different directory?
I think we could instead clone the repository in a temporary directory, something like:
```bash
TMPDIR="$(mktemp -d)"

git clone https://github.com/sqlpad/sqlpad.git "${TMPDIR}/sqlpad"
```

Then do:
```bash
helm upgrade --install --namespace sqlpad sqlpad \
    --create-namespace \
    --set image.repository=$IMAGE_REGISTRY_REPO \
    --set image.tag=$IMAGE_TAG \
    "${TMPDIR}/sqlpad/sqlpad-charts"
```

Reply:
That was a great call out, thanks, just attempted to do it.